### DISCUSSION TITLE
Altitude controller and hdg with pmdg's B77W (msfs) not working

### DISCUSSION THREAD

**Discussion by Niko7133** [nguyenquyhy/Flight-Tracker-StreamDeck#426] [Q&A]
Hi guys, I am trying to use my streamdeck plus with the B77W, too bad that when I go to change the heading and altitude, on the streamdeck it changes but not on the simulator. Instead with the other planes they work. Do you know how I can fix it

**Comment by nguyenquyhy** [comment]
The only way would be to ask PMDG to make the plane compatible with SimConnect.
